src/main/java/org/wildfly/security/x500/X500.java

```diff
@@ -18,8 +18,14 @@
 
 package org.wildfly.security.x500;
 
+import org.wildfly.security._private.ElytronMessages;
+
 import java.security.cert.X509Certificate;
 import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Properties;
```

Comment:
Unless we can establish that the names we are using come from and are consumed specifically by things conforming to X.500.  Ideally the classifications we'd use would be by specification rather than by use case, if possible.